contrib/shotman/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "shotman"
+pkgver = "0.4.5"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo", "pkgconf", "scdoc"]
+makedepends = ["libxkbcommon-devel", "rust-std"]
+depends = ["slurp"]
+pkgdesc = "Uncompromising screenshot GUI for Wayland"
+maintainer = "triallax <triallax@tutanota.com>"
+license = "ISC"
+url = "https://sr.ht/~whynothugo/shotman"
+source = f"https://git.sr.ht/~whynothugo/shotman/archive/v{pkgver}.tar.gz"
+sha256 = "59ade23793294b5021d4aa6e4151cd3dc2063079011c67ab12e6c6b9d4031b2e"
+env = {
+    "SHOTMAN_VERSION": f"v{pkgver}",
+}
+# No tests
+options = ["!check"]
+
+
+def post_build(self):
+    self.do("bmake", "shotman.1")
+
+
+def post_install(self):
+    self.install_license("LICENCE.md")
+    self.install_man("shotman.1")
```

Comment:
The package also has completions, but they're generated by running a binary, which won't work in cross. Not sure what to do about that.

Reply:
you could make it !cross like sbctl

Reply:
Is it worth making it !cross just for completions? We could generate them only on native builds, but idk

Reply:
!cross is fine